core/src/main/java/forklift/consumer/Consumer.java

```diff
@@ -401,7 +402,9 @@ private final void configureConstructorInjection() {
                 fields.get(clazz).forEach(field -> {
                     log.trace("Inject target> Field: ({})  Decorator: ({})", field, decorator);
                     try {
-                        Object value = getInjectableValue(field.getAnnotation(decorator), field.getName(), clazz, msg);
+                        String fieldName = Optional.ofNullable(field.getAnnotation(Named.class))
+                                .map(a -> a.value()).orElse(field.getName());
```

Comment:
do you want orElse or orElseGet?  orElse is always executed where as orElseGet is not